.github/workflows/ci-cd-workflow.yml

```diff
@@ -203,7 +203,10 @@ jobs:
           aws-region: eu-west-2
 
       - name: Deploy Amazon ECS task definition
-        run: aws ecs update-service --cluster trainee-preprod --service ${{ github.event.repository.name }} --task-definition ${{ github.event.repository.name }}
+        run: aws ecs update-service \
+          --cluster revalidation-preprod \
```

Comment:
@Judge40 I've made this change as I am wondering should the cluster be 'revalidation-preprod' instead of 'trainee-preprod' for this deploy-reval step.